app/src/main/java/io/github/djunicode/salespersontracker/TaskRetriever.java

```diff
@@ -0,0 +1,12 @@
+package io.github.djunicode.salespersontracker;
+import java.util.List;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Headers;
+
+public interface TaskRetriever {
+    String BASE_URL = "http://127.0.0.1:8000/Operations/";
+    @Headers("Authorization: Token e0b7118f129d7f81bf440df8846d6d1d979c8709")
```

Comment:
I am assuming this was hardcoded for the purpose of testing, But you should've accessed from a persistence like SharedPreferences